hw_4/task_3.py

```diff
@@ -0,0 +1,15 @@
+# Напишите функцию принимающую на вход только ключевые параметры и возвращающую словарь,
+# где ключ — значение переданного аргумента, а значение — имя аргумента. Если ключ не хешируем,
+# используйте его строковое представление.
+
+def get_dict(**kwargs):
+    res = {}
+    for key, value in kwargs.items():
+        if not hash(key):
+            res[str(value)] = key
+        else:
+            res[value] = key
+    return res
+
+
+print(get_dict(a=55, b='ttt', c=True))
```

Comment:
Попробуйте запустить функцию с такими аргументами
```suggestion
print(get_dict(a=55, b=['ttt'], c=True))
```

Reply:
После этого исправьте код так, чтобы ошибки не возникало. При этом постарайтесь не использовать обработку исключений.